dummydata.js

```diff
@@ -0,0 +1,55 @@
+const AWS = require('aws-sdk');
```

Comment:
What is the purpose of this file? If it's not needed, delete it. Also, rename it to be more descriptive of the needs